docs/docusaurus/versioned_docs/version-0.18/cloud/deploy_gx_agent.md

```diff
@@ -0,0 +1,247 @@
+
+If you've used GX Cloud previously, you have your access token and organization ID, and you need to restart the GX Agent, see [Deploy the GX Agent](#deploy-the-gx-agent).
+
+<Tabs
+  groupId="copy-token"
+  defaultValue='new'
+  values={[
+  {label: 'New GX Cloud account', value:'new'},
+  {label: 'Existing GX Cloud account', value:'existing'},
+  ]}>
+<TabItem value="new">
+
+1. Sign in to GX Cloud.
+
+2. Complete the survey and then click **Continue to GX Cloud**.
+
+3. Copy and then paste the **Access token** and **Organization ID** values into a temporary file. You'll need them to deploy the GX Agent.
+
+4. Click [**Deploy the GX Agent**](#deploy-the-gx-agent).
+
+</TabItem>
+<TabItem value="existing">
+
+Use the information provided here to view your organization ID or create a new access token. This can be helpful if you've forgotten your organization ID or access token, and you need to restart the GX Agent.
+
+1. In GX Cloud, click **Settings** > **Tokens**.
+
+2. In the **User access tokens** pane, click **Create user access token**.
+
+3. In the **Token name** field, enter a name for the token that will help you quickly identify it.
+
+4. Click **Create**.
+
+5. Copy and then paste the user access token into a temporary file. The token can't be retrieved after you close the dialog.
+
+6. Click **Close**.
+
+7. Copy the value in the **Organization ID** field into the temporary file with your user access token and then save the file. 
+
+    GX recommends deleting the temporary file after you set the environment variables.
+
+8. [Deploy the GX Agent](#deploy-the-gx-agent).
+
+</TabItem>
+</Tabs>
+
+## Deploy the GX Agent
+
+The GX Agent runs open source GX code in GX Cloud, and it allows you to securely access your data without connecting to it or interacting with it directly. To learn more about the GX Agent and deployment patterns, see [Deploy the GX Agent](#deploy-the-gx-agent).
+
+### Self-hosted and org-hosted deployments
+
+To try GX Cloud, you use a [self-hosted deployment](./about_gx#self-hosted-deployment-pattern) to run the GX Agent with Docker, connect the GX Agent to your target Data Sources, and use the GX Cloud web UI to define your Data Assets, create Expectations, and run Validations. A self-hosted deployment is recommended when you want to test GX Cloud features and functionality, and it differs from the recommended [org-hosted deployment](./about_gx.md#org-hosted-deployment-pattern), in which the GX Agent runs in your organization's deployment environment.
+
+### Prerequisites
+
+- You have a GX Cloud access token and organization ID. See [Get your access token and organization ID](#get-your-access-token-and-organization-id). 
+
+- You have a [Docker instance](https://docs.docker.com/get-docker/) or [kubectl](https://kubernetes.io/docs/tasks/tools/).
+
+<Tabs
+  groupId="deploy-agent"
+  defaultValue='local'
+  values={[
+  {label: 'Self-hosted', value:'local'},
```

Comment:
Since we want to encourage the org-hosted production deployment, can we switch the order of the tabs or have the org-hosted tab selected by default?

Reply:
@annabarr, I updated the topic so that org-hosted appears first.